### PR TITLE
Switch to imlteam/copr

### PR DESCRIPTION
**Pull request by jgrund**
Switch to using the docker builder for this repo.

A black check was added. This will fail in this patch, but I'll fix in a
follow-on.

Signed-off-by: Joe Grund <jgrund@whamcloud.io>

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -1,28 +1,36 @@
...
-    - ./include/travis/run_in_centos7_docker.sh include/travis/mock_build_test.sh
+    - export SPEC=python-iml-common.spec
+    - docker run -it -e SPEC="$SPEC" -e LOCAL_ONLY="True" -v $(pwd):/build:rw imlteam/copr
+    - ((`find _topdir/RPMS -name *.rpm | wc -l` > 0))
+  - stage: cd
+    name: "Continuous Deployment"
+    script:
+      - export OWNER=managerforlustre
+      - export PROJECT=manager-for-lustre-devel
+      - export PACKAGE=python-iml-common1.4
```
Comment (johnsonw) [inline .travis.yml]: Is the 1.4 to save for an older version? I don't understand why the version number is in the package name.

File: .travis.yml [hunk 2]
```diff
@@ -34,14 +42,18 @@ jobs:
...
-      skip_cleanup: true
-      provider: script
-      script: ./travis_wait "./include/travis/run_in_centos7_docker.sh include/travis/copr-deploy.sh build"
-      on:
-        all_branches: true
+    name: "Copr deploy"
+    script:
+        - export OWNER=managerforlustre
+        - export PROJECT=manager-for-lustre
+        - export PACKAGE=python-iml-common1.4
```
Comment (johnsonw) [inline .travis.yml]: 1.4?

File: .travis.yml (hunk 1 above)
Comment (jgrund) [reply .travis.yml]: That's just how this one was done originally.

File: .travis.yml (hunk 2 above)
Comment (jgrund) [reply .travis.yml]: That's just how this one was done originally.

**Pull request merged**